### DISCUSSION TITLE
CGAL Mesh boolean difference and normals

### DISCUSSION THREAD

**Discussion by petrasvestartas** [CGAL/cgal#6962] [Q&A]
After Mesh boolean difference, does the output mesh has by default computed vertex normals?
If yes, how can I retrieve them?

**Comment by MaelRL** [comment]
There are no default computed vertex normals, you have to construct them yourself using for example [PMP::compute_vertex_normals()](https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__normal__grp.html#ga74e6b247d7e28beb677076aad5614d8c).
